deploy/auxiliary/caf-secure-hubSpoke.json

```
@@ -0,0 +1,244 @@
+                ],
+                "enableDdosProtection": "[if(equals(parameters('enableDdoS'), 'Yes'), 'true', 'false')]",
+                "ddosProtectionPlan": {
+                    "id": "[if(equals(parameters('enableDdoS'), 'Yes'), resourceId('Microsoft.Network/ddosProtectionPlans', variables('dDoSName')), json('null'))]"
+                }
+            }
+        },
+
+        // Spoke one network
+        {
+            "type": "Microsoft.Network/virtualNetworks",
+            "apiVersion": "2020-05-01",
+            "name": "[variables('spokeNetworkOne').name]",
+            "location": "[variables('location')]",
+            "properties": {
+                "addressSpace": {
+                    "addressPrefixes": [
+                        "[variables('spokeNetworkOne').addressPrefix]"
+                    ]
+                },
+                "subnets": [
+                    {
+                        "name": "[variables('spokeNetworkOne').subnetName]",
+                        "properties": {
+                            "addressPrefix": "[variables('spokeNetworkOne').subnetPrefix]"
+                        }
+                    }
+                ]
+            }
+        },
+
+        // Spoke two network
+        {
+            "type": "Microsoft.Network/virtualNetworks",
+            "apiVersion": "2020-05-01",
+            "name": "[variables('spokeNetworkTwo').name]",
+            "location": "[variables('location')]",
+            "properties": {
+                "addressSpace": {
+                    "addressPrefixes": [
+                        "[variables('spokeNetworkTwo').addressPrefix]"
+                    ]
+                },
+                "subnets": [
+                    {
+                        "name": "[variables('spokeNetworkTwo').subnetName]",
+                        "properties": {
+                            "addressPrefix": "[variables('spokeNetworkTwo').subnetPrefix]"
+                        }
+                    }
+                ]
+            }
+        },
+
+        // Virtual network peering spoke one to hub
+        {
+            "type": "Microsoft.Network/virtualNetworks/virtualNetworkPeerings",
+            "apiVersion": "2020-05-01",
+            "name": "[concat(variables('spokeNetworkOne').name, '/peer-to-', variables('hubNetwork').name)]",
+            "location": "[variables('location')]",
+            "dependsOn": [
+                "[resourceId('Microsoft.Network/virtualNetworks', variables('hubNetwork').name)]",
+                "[resourceId('Microsoft.Network/virtualNetworks', variables('spokeNetworkOne').name)]"
+            ],
+            "properties": {
+                "remoteVirtualNetwork": {
+                    "id": "[resourceId('Microsoft.Network/virtualNetworks', variables('hubNetwork').name)]"
+                },
+                "allowVirtualNetworkAccess": true,
+                "allowForwardedTraffic": true,
+                "allowGatewayTransit": true,
```

```suggestion
                "allowGatewayTransit": false,
```
Typically only needed in hub to spoke peering, and only to support on-prem VPN Gateway/ER.

---

Done